src/current/v24.2/ui-overload-dashboard.md

```diff
@@ -15,93 +21,113 @@ To view this dashboard, [access the DB Console]({% link {{ page.version.version
+This graph shows the 99th percentile latency of requests waiting in the Admission Control store queue, as tracked by the `admission.wait_durations.kv-stores-p99` and the `admission.wait_durations.elastic-stores-p99` metrics. There are separate lines for KV write and elastic (background) write.
 
-- In the node view, the graph shows the number of microseconds per second that there were no write tokens on the selected node.
-- In the cluster view, the graph shows the number of microseconds per second that there were no write tokens across all nodes in the cluster.
+- In the node view, the graph shows the delay duration of KV write and elastic (background) write on the selected node.
+- In the cluster view, the graph shows the delay duration of KV write and elastic (background) write across all nodes in the cluster.
 
-## Flow Tokens Wait Time: 75th percentile
+## Admission Queueing Delay p99 – Background (Elastic) CPU
 
-This graph shows the 75th [percentile](https://wikipedia.org/wiki/Percentile#The_normal_distribution_and_percentiles) of latency for regular requests and elastic requests spent waiting for flow tokens, as tracked respectively by the `cr.node.kvadmission.flow_controller.regular_wait_duration-p75` and the `cr.node.kvadmission.flow_controller.elastic_wait_duration-p75` metrics. There are separate lines for regular flow token wait time and elastic flow token wait time.
+This graph shows the 99th percentile latency of requests waiting in the Admission Control elastic CPU queue, as tracked by the `admission.wait_durations.elastic-cpu-p99` metric.
 
-- In the node view, the graph shows the 75th percentile of latency for regular requests and elastic requests spent waiting for flow tokens on the selected node.
-- In the cluster view, the graph shows the 75th percentile of latency for regular requests and elastic requests spent waiting for flow tokens across all nodes in the cluster.
+- In the node view, the graph shows the delay duration of KV write on the selected node.
+- In the cluster view, the graph shows the delay duration of KV write across all nodes in the cluster.
 
-## Requests Waiting For Flow Tokens
+## Admission Queueing Delay p99 – Replication Admission Control
 
-This graph shows the number of regular requests and elastic requests waiting for flow tokens, as tracked respectively by the `cr.node.kvadmission.flow_controller.regular_requests_waiting` and the `cr.node.kvadmission.flow_controller.elastic_requests_waiting` metrics. There are separate lines for regular requests waiting and elastic requests waiting.
+This graph shows the 99th percentile latency of requests waiting in the Replication Admission Control queue, as tracked by the `kvadmission.flow_controller.regular_wait_duration-p99` and the `kvadmission.flow_controller.elastic_wait_duration-p99` metrics. There are separate lines for regular flow token wait time and elastic (background) flow token wait time. These metrics are indicative of store overload on replicas.
```

Comment:
same as above re: adding link to 'operations in AC' subsection of AC overview page